app/digitizingcontroller.cpp

```diff
@@ -193,3 +193,23 @@ QgsQuickFeatureLayerPair DigitizingController::lineFeature()
 
   return QgsQuickFeatureLayerPair(f, featureLayerPair().layer());
 }
+
+QgsPoint DigitizingController::mapCoords(QgsQuickFeatureLayerPair pair)
```

Comment:
Please rename to make it clear this is meant to work with point features - e.g. pointFeatureMapCoordinates